requirements/pip.in

```diff
@@ -0,0 +1,5 @@
+# Core dependencies for installing other packages
+
```

Comment:
you didn't run the `make upgrade`? I don't see anything listed in `pip.txt`

Reply:
Ran Make upgrade again to update the file.